zebROS_ws/src/field_obj_tracker/include/field_obj_tracker/objtype.hpp

```diff
@@ -55,9 +55,9 @@ class ObjectType
 		bool operator== (const ObjectType &t1) const;
 
 		//coordinate conversions
-		cv::Point3f screenToWorldCoords(const cv::Rect &screen_position, double avg_depth, const image_geometry::PinholeCameraModel &model) const;
+		cv::Point3f screenToWorldCoords(const cv::Rect &screen_position, double avg_depth, const sensor_msgs::CameraInfo camera_info_) const;
```

Comment:
For both of these, changing them to take a const reference would be a slight optimization (it won't have to make a copy of the entire CameraInfo struct with the reference case).
Syntax is `sensor_msgs::CameraInfo &camera_info_`, that is, just adding an & in front of the parameter name.

And nitpicking on my part - remove the _ from the var name. We typically use _ to refer to member vars, and this is just a local parameter so it should be without the _